## Wednesday {{date}}

- [ ] 0830AM [NPPS Coffee](https://bnl.zoomgov.com/j/16157150845?pwd=NXNqTi9ZWEFBKzYwRXQ5U3NXU1dBZz09)
- [ ] 1000AM [sPHENIX Distributed Computing](https://bnl.zoomgov.com/j/16157150845?pwd=NXNqTi9ZWEFBKzYwRXQ5U3NXU1dBZz09)
- [ ] 1200PM [STAR S&C Meeting](https://lbnl.zoom.us/j/97026562983?pwd=VGVXbzhYUUhheEJ2cFMyVVdVRXowZz09)

----------------------------------------

- Shrek Update

- Problem with submitting the sPHENIX workflow could be related to how the output data set is defined in the CWL.  For some reason,... it is not pass2/outDS but just pass2?
	- YAML files need to be adjusted so that the output data sets have the proper naming convention for PanDA...
	- This is really a TODO list item... we should be automating the naming of the data sets behind the scenes... users can name them what they want, but internally they should be represented for PanDA.
	- The input data set for the second stage was also ill defined.
	- ODDBALL.... pythia8charm has no optargs defined.
		- b/c I failed to specify any output files!  **This is an untrapped failure mode.**

So.  After taking care to properly define the jobs...
https://panda-doma.cern.ch/tasks/?jeditaskid=65578

sPHENIX pass1 is running... I hope.

- I am not seeing output root files anywhere... so... run local...

input_line_270:2:2: error: no matching function for call to 'Fun4All_G4_HF_pp_signal'
 Fun4All_G4_HF_pp_signal(100,"Charm","G4Hits-Charm-1234567890-12345.root","",".") /* '.x' tries to invoke a function with the same name as the macro */
 ^~~~~~~~~~~~~~~~~~~~~~~
/direct/sphenix+u/jwebb2/work/2022/SHREK/shrek/.shrek-submit-MCTEST-9/Fun4All_G4_HF_pp_signal.C:42:5: note: candidate function not viable: no known conversion from 'const char [2]' to 'const int' for 5th argument
int Fun4All_G4_HF_pp_signal(

... bah.  Had the parameter list wrong.


TODO:
- Jinlong has cycled back... still having embedding issues.  Follow up on this.
- Still owe Diptanil time... shift to this later today.